molo/usermetadata/templates/persona.html

```diff
@@ -2,18 +2,23 @@
 {% load wagtailimages_tags i18n %}
 
 {% block content %}
-<div>
-    <ul>
+<div class="personae">
+    <h3 class="personae__title">Are you a ...</h3>
```

Comment:
perhaps rename this to `heading heading--personae`, and place it outside `div.personae`